### PR TITLE
Notebooks: Hide link and image buttons in text cell toolbar in WYSIWYG mode

### PR DESCRIPTION
**Pull request by lucyzhang929**

### CODE REVIEW

**Review by halerankin — APPROVED** [review]
Nicely done!

File: src/sql/workbench/contrib/notebook/browser/cellViews/markdownToolbar.component.ts [hunk 1]
```diff
@@ -42,6 +42,17 @@ export class MarkdownToolbarComponent {
 	public splitViewButton = localize('splitViewButton', "View as Split");
 	public markdownButton = localize('markdownButton', "View as Markdown");
 
+	private boldButton;
```
Comment (chlafreniere) [inline src/sql/workbench/contrib/notebook/browser/cellViews/markdownToolbar.component.ts:45]: let's include types here if possible

Comment (chlafreniere) [reply src/sql/workbench/contrib/notebook/browser/cellViews/markdownToolbar.component.ts:45]: and _ should precede the names

File: src/sql/workbench/contrib/notebook/browser/cellViews/markdownToolbar.component.ts [hunk 2]
```diff
@@ -95,20 +106,71 @@ export class MarkdownToolbarComponent {
 			this.optionParagraph,
 			undefined
 		);
-		dropdownMenuActionViewItem.render(buttonDropdownContainer);
+		dropdownMenuActionViewItem.render(this.buttonDropdownContainer);
 		dropdownMenuActionViewItem.setActionContext(this);
 
+		// Hide link and image buttons in WYSIWYG mode
+		if (this.cellModel.showPreview && !this.cellModel.showMarkdown) {
```
Comment (chlafreniere) [inline src/sql/workbench/contrib/notebook/browser/cellViews/markdownToolbar.component.ts:113]: I'm worried about duplication here.

How can we bring this # down of declaring these arrays?

**Review by chlafreniere — COMMENTED** [review]
Let's see if we can reuse code a bit.

**Review by chlafreniere — APPROVED** [review]

**Pull request merged**